### PR TITLE
Friday the Thirteenth Bugfixes

### LINKED ISSUE #11713: Spray Webbing
Putting a spray bottle into your webbing causes you to spray it. Tested with a bottle of Sterilizine.

### LINKED ISSUE #12204: Equiping 'webbing' when wearing 'electronic jumpsuit' causes the item to glitch
webbing

I play an engineer, and I tried out the 'electronic jumpsuit' loadout item. When I have this equipped, trying to equip a 'webbing' or 'brown webbing vest' makes the item glitch out. 

As you can see in the screenshot, my right hand holds a 'webbing' item, and my left hand a 'brown webbing vest'. After pressing 'E' to equip them, my hands are, for all intents and purposes, empty. The webbing wasn't equipped, and you can still see it in my hand, but I am now once again able to pick up new items and interact with them. You see this in the screenshot, my right hand contains a pill, that was just taken from the bottle in the left hand.

In short, I expect to be able to equip webbing items as normal, and not have them disappear while still showing up in my hand slots.

### LINKED ISSUE #14056: Smoke enters beakers/bottles even when lidded and containers inside machines.
Example: Scrubber event in chemistry. Lidded beakers and bottles all around. The smoke enters them and the chem is added inside.
It also enters the contains inside the chemmasters.
It does not enter the containers inside of the chem dispenser.

### PR DESCRIPTION
**Pull request by alsoandanswer**
Title

Fixes #14056
Fixes #12204
Fixes #11713

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!bugfix

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/effects/chem/chemsmoke.dm [hunk 1]
```diff
@@ -48,13 +48,19 @@
 /obj/effect/effect/smoke/chem/Crossed(atom/movable/AM)
 	..()
 	if(!istype(AM, /obj/effect/effect/smoke/chem))
-		reagents.splash(AM, splash_amount, copy = 1)
+		if(istype(AM, /obj/item/reagent_containers) && !AM.is_open_container())
+			return
+		else
+			reagents.splash(AM, splash_amount, copy = 1)
```
Comment (SleepyGemmy) [inline code/game/objects/effects/chem/chemsmoke.dm:54]:
```suggestion
			reagents.splash(AM, splash_amount, copy = TRUE)
```

File: code/game/objects/effects/chem/chemsmoke.dm [hunk 2]
```diff
@@ -48,13 +48,19 @@
...
 
 /obj/effect/effect/smoke/chem/proc/initial_splash()
 	for(var/turf/T in view(1, src))
 		for(var/atom/movable/AM in T)
 			if(!istype(AM, /obj/effect/effect/smoke/chem))
-				reagents.splash(AM, splash_amount, copy = 1)
+				if(istype(AM, /obj/item/reagent_containers) && !AM.is_open_container())
+					return
+				else
+					reagents.splash(AM, splash_amount, copy = 1)
```
Comment (SleepyGemmy) [inline code/game/objects/effects/chem/chemsmoke.dm:63]:
```suggestion
					reagents.splash(AM, splash_amount, copy = TRUE)
```

**Review by SleepyGemmy — COMMENTED** [review]
[nvm]

**Review by SleepyGemmy — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**